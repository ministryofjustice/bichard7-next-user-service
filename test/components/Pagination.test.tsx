import { render } from "@testing-library/react"
import Pagination from "components/Pagination"

it("should render the component and match the snapshot", () => {
  const { container } = render(<Pagination href="/dummy-url" maxItemsPerPage={10} pageNumber={1} totalItems={100} />)

  expect(container).toMatchSnapshot()
})
