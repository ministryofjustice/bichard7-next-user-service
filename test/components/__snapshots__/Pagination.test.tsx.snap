// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component and match the snapshot when total items is more than zero 1`] = `
<div>
  <div
    class="govuk-hint"
  >
    <a
      class="govuk-link"
      data-test="Prev"
      href="/dummy-url?page=0"
    >
      &lt; Prev
    </a>
    <span
      style="padding: 10px;"
    >
      Page 2 of 10
    </span>
    <a
      class="govuk-link"
      data-test="Next"
      href="/dummy-url?page=2"
    >
      Next &gt;
    </a>
  </div>
</div>
`;

exports[`should render the component and match the snapshot when total items is zero 1`] = `<div />`;
